docs/howto/operate/new-cluster/aws.md

```diff
@@ -156,8 +157,8 @@ have least amount of permissions possible.
    they are accessible from our automatic deployment.
 
    ```bash
-   terraform output -raw continuous_deployer_creds > ../../config/clusters/<your-cluster-name>/deployer-credentials.secret.json
-   sops --output config/clusters/<your-cluster-name>/enc-deployer-credentials.secret.json --encrypt ../../config/clusters/<your-cluster-name>/deployer-credentials.secret.json
+   terraform output -raw continuous_deployer_creds > ../../config/clusters/<your-cluster-name>/enc-deployer-credentials.secret.json
+   sops --in-place --encrypt ../../config/clusters/<your-cluster-name>/enc-deployer-credentials.secret.json
    ```
```

Comment:
Maybe we can keep the original command and provide your suggestion as an alternative? I like to not use `--in-place` and explicit add the `enc-` through `--output` because our `.gitignore` file prevents files that contain `secret` in their name but are not prefixed with `enc-` from being accidentally committed to the repo. I can then clean up unencrypted files using `git clean`.

Reply:
@sgibson91 sorry Sarah, I know this isn't the first time I'd changed this to `--in-place` - I had forgotten the last time you had reminded me of it! I've just reverted this change to go back to using `--output` instead as it was before, and will keep this in mind for the future!